src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', () => {
```

Comment:
You need to pass the `event` object as a parameter to the event listener function: `gallery.addEventListener('click', (event) => { ... });`. This ensures that the `event` object is properly scoped and accessible within the function.